modules/assim.sequential/R/Analysis_sda_block.R

```diff
@@ -130,6 +130,10 @@ build.block.xy <- function(settings, block.list.all, X, obs.mean, obs.cov, t) {
   }
   
   #Handle observation
+  #observation number per site
+  obs_per_site <- obs.mean[[t]] %>% 
+    purrr::map(function(site.obs){length(site.obs)}) %>% 
+    unlist()
```

Comment:
This can be simplified to
```suggestion
  obs_per_site <- purrr::map_int(obs.mean[[t]], length)
```

Reply:
Fixed, thank you!